src/reanimated2/layoutReanimation/web/animation/Bounce.web.ts

```diff
@@ -0,0 +1,165 @@
+import { parseAnimationObjectToKeyframe } from '../animationParser';
```

Comment:
Great idea with splitting animation configs across several files depending on the type. Thanks, I appreciate it!